node/_crypto/randomFill.ts

```diff
@@ -0,0 +1,81 @@
+import randomBytes, { MAX_SIZE as kMaxUint32 } from "./randomBytes.ts";
+import { Buffer } from "../buffer.ts";
+
+const kBufferMaxLength = 0x7fffffff;
+
+function assertOffset(offset: number, length: number) {
+  if (offset > kMaxUint32 || offset < 0) {
+    throw new TypeError("offset must be a uint32");
+  }
+
+  if (offset > kBufferMaxLength || offset > length) {
+    throw new RangeError("offset out of range");
+  }
+}
+
+function assertSize(size: number, offset: number, length: number) {
+  if (size > kMaxUint32 || size < 0) {
+    throw new TypeError("size must be a uint32");
+  }
+
+  if (size + offset > length || size > kBufferMaxLength) {
+    throw new RangeError("buffer too small");
+  }
+}
+
+export default function randomFill(
+  buf: Buffer,
+  cb: (err: Error | null, buf?: Buffer) => void,
+): void;
+
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  cb: ((err: Error | null, buf?: Buffer) => void),
+): void;
+
+export default function randomFill(
+  buf: Buffer,
+  offset: number,
+  size: number,
+  cb: (err: Error | null, buf?: Buffer) => void,
```

Comment:
DefinitelyTyped gives this `buf` type as non-optional. ref: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/9ddc52c6175556e3a9a0dce6234c1713fdcdccae/types/node/crypto.d.ts#L1911

I think that makes more sense because the buffer is already given as a required parameter. So it should be always available.